### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by johnykvsky** [auraphp/Aura.Input#77]
Hi, will it ever happen, or it's a dead project? If it's about waiting for connection to FilterInterface for multidimensional arrays - it's taking so long, that maybe we can assume it will never happen and just release new version as a bug/compatibility fix?

**Comment by harikt** [comment]
Yes, I understand your concern. I got a bit busy and missed your message. We are in the process of releasing 6.x . But there is a BC break . Just a note.
